back/boxtribute_server/cron/data_faking.py

```diff
@@ -0,0 +1,838 @@
+from faker import Faker, providers
+from flask import g
+from peewee import fn
+
+from ..auth import CurrentUser
+from ..business_logic.beneficiary.crud import (
+    create_beneficiary,
+    create_transaction,
+    deactivate_beneficiary,
+    update_beneficiary,
+)
+from ..business_logic.box_transfer.agreement.crud import (
+    accept_transfer_agreement,
+    cancel_transfer_agreement,
+    create_transfer_agreement,
+    reject_transfer_agreement,
+)
+from ..business_logic.box_transfer.shipment.crud import (
+    _retrieve_shipment_details,
+    cancel_shipment,
+    create_shipment,
+    mark_shipment_as_lost,
+    send_shipment,
+    start_receiving_shipment,
+    update_shipment_when_preparing,
+    update_shipment_when_receiving,
+)
+from ..business_logic.tag.crud import create_tag, delete_tag, update_tag
+from ..business_logic.warehouse.box.crud import create_box, update_box
+from ..business_logic.warehouse.location.crud import create_location, delete_location
+from ..business_logic.warehouse.product.crud import (
+    create_custom_product,
+    delete_product,
+    edit_custom_product,
+)
+from ..business_logic.warehouse.qr_code.crud import create_qr_code
+from ..db import db
+from ..enums import (
+    BoxState,
+    HumanGender,
+    Language,
+    ProductGender,
+    TagType,
+    TransferAgreementType,
+)
+from ..models.definitions.base import Base
+from ..models.definitions.box import Box
+from ..models.definitions.product import Product
+from ..models.definitions.size import Size
+from ..models.utils import convert_ids
+
+NR_BASES = 4
+NR_OF_CREATED_TAGS_PER_BASE = 20
+NR_OF_DELETED_TAGS_PER_BASE = round(NR_OF_CREATED_TAGS_PER_BASE / 10)
+LOCATION_NAMES = ("Stockroom", "WH", "FreeShop", "LOST", "SCRAP", "Unused WH")
+NR_OF_CREATED_LOCATIONS_PER_BASE = len(LOCATION_NAMES)
+NR_OF_ADULTS_PER_BASE = 100
```

Comment:
pls create only one base with 300 beneficiaries (load testing) and the rest with less than half the amount.